src/dotnet/Program.cs

```diff
@@ -114,6 +114,8 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 }
                 else
                 {
+                    firstTimeUseSentinelPath = ConfigureDotNetForFirstTimeUse();
```

Comment:
@blackdwarf When I chatted with you last week, you said it was ok to use a single environment variable for both scenarios right now, given that they are working hand in hand. What changed that this is no longer good? We are in a time crunch right now, unless we have a very good reason to change this, I would like to push back.

Also, my question was wether it is correct to have telemetry turned off until the user sees the EULA.